sci-libs/proj/proj-5.0.1.recipe

```diff
@@ -56,10 +60,21 @@ BUILD_PREREQUIRES="
 	cmd:make
 	"
 
+defineDebugInfoPackage proj$secondaryArchSuffix \
+	"$commandBinDir"/cct \
+	"$commandBinDir"/cs2cs \
+	"$commandBinDir"/geod \
+	"$commandBinDir"/gie \
+	"$commandBinDir"/nad2bin \
+	"$commandBinDir"/proj \
+	$libDir/libproj.so.$libVersion
+
 BUILD()
 {
 	autoreconf -fi
-	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
+	runConfigure --omit-dirs binDir ./configure \
+		--bindir=$commandBinDir
+
 	make $jobArgs
```

Comment:
Hmm, I wouln't  change `BUILD`.
```
	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
```
is OK.
That said, if you really wish to change it, it would be nice to also replace `--bindir=$commandBinDir` by `--bindir="$commandBinDir"`.